watson-developer-cloud/discovery-components
# chore: increase unit test coverage for CIDocument filterUtils

#### What do these changes do/fix?
Adds on to the unit test for the CIDocument utils file `filterUtils.ts` for higher test coverage of our code.

https://zenhub.ibm.com/app/workspaces/discovery-unified-5728bf39d2c20f776e05d0ef/issues/watson-discovery/docviz-squad-issue-tracker/234

#### How do you test/verify these changes?
`yarn test watch:filterUtils` and see that tests pass.

Run `yarn test:coverage` and view the coverage of the CIDocument/utils/filterUtils.ts file to see the results are statements: 91.67%, branches: 90.91%, and functions: 100%.
